lib/fetch/request.js

```diff
@@ -138,8 +138,47 @@ class Request {
 
     // 12. Set request to a new request with the following properties:
     request = makeRequest({
```

Comment:
by listing all the properties, I assumed there are some that aren't meant to be copied to the new request. It makes sense as previously origin and urlList were not supposed to be copied, but were being done by the lib.